pkts-core/src/main/java/io/pkts/framer/RTPFramer.java

```diff
@@ -98,9 +98,7 @@ public RtpPacket frame(final TransportPacket parent, final Buffer buffer) throws
             final int csrcCount = b & 0x0F;
 
             if (hasExtension) {
-                final short extensionHeaders = buffer.readShort();
```

Comment:
The buffer.readShort must take place since it advances the buffer. Please keep because just having buffer.readShort() without the assignment will look confusing. Of course, you could add a comment something like so instead:

// read the extension headers, which we are currently ignoring
buffer.readShort()